### PR TITLE
feat(configure axios): configure axios to call api 

### PR DESCRIPTION
**Pull request by munyanezaarmel**
### What does this PR do?
this PR set up axios which will be used when calling backend api

### How should this be manually tested?
`clone this repo cd then run yarn start `
### What are the relevant pivotal tracker stories?
[#182906952](https://www.pivotaltracker.com/story/show/182906952)

### CODE REVIEW

**Review by kalisaNkevin — APPROVED** [review]

File: utils/api.js [hunk 1]
```diff
@@ -0,0 +1,8 @@
+import axios from "axios";
+export default axios.create({
+    baseURL:"https://elite-bn-be.herokuapp.com/",
+    headers: {
+        'Authorization': 'Bearer ' + localStorage.getItem('token')
+      }
```
Comment (ericus123) [inline utils/api.js]: define the token instead.
token = localStorage.getItem('token');
then concat it with the variable with `"Bearer"` instead

File: utils/api.js [hunk 2]
```diff
@@ -0,0 +1,8 @@
+import axios from "axios";
+export default axios.create({
+    baseURL:"https://elite-bn-be.herokuapp.com/",
```
Comment (ericus123) [inline utils/api.js]: this url should be from an environment variable.
also remove the `/` it will be added by the person writting an api call

**Review by ericus123 — CHANGES_REQUESTED** [review]

File: utils/api.js (hunk 2 above)
Comment (munyanezaarmel) [reply utils/api.js]: done , i added `/` to test if it is working

File: utils/api.js (hunk 1 above)
Comment (munyanezaarmel) [reply utils/api.js]: done

**Review by ericus123 — APPROVED** [review]

**Review by ericus123 — APPROVED** [review]

**Pull request merged**